### ISSUE TITLE
set_model_weights after batch norm recalibration?

### ISSUE THREAD

**Issue by woojinchokimm** [simon-larsson/keras-swa#11]
Hello, first off, thank you for this swa implementation - it's been extremely useful to avoid TF addons :D

Are there any thoughts on modifying the callback to function with EarlyStopping?
